### DISCUSSION TITLE
FYSETC Spider V3.0 - Voron 2.4 wiring diagram

### DISCUSSION THREAD

**Discussion by EKVoron24** [FYSETC/FYSETC-SPIDER#153] [Q&A] [answered]
I have sourced a wiring diagram for the Voron 2.4 using a FYSETC Spider V3.0 board at https://github.com/FYSETC/FYSETC-Voron-2/blob/main/VORON2.4R2_SPIDER_V30_Wiring.pdf however this uses a separate 'BL-Touch' board to wire in the hot end that I don't have.
Does anyone know where to find the wiring diagram for the Spider V3.0 board that wires the hotend and associated hardware directly to the board.

**Comment by cyberjak2k** [comment]
There is a pdf that shows BLtouch connection and I recall having to trace
the schematics a bit back to find the pin name for the Sig pin but it's
just above where the Raspberry pi goes and there's silk screening on the
board showing this just not the pin names. If you struggle I'll look it up
again.
But I don't think there's a pdf with each scenario for example I went with
a TAP

**Reply by EKVoron24** [reply]
Thanks for the reply but I'm looking for a wiring diagram that doesn't have the BL-Touch as I don't have a BL-Touch so I need to wire the top end stuff directly to the Spider board not via a BL-Touch (if that makes sense).

**Comment by cyberjak2k** [comment, marked as answer]
So are you using just a Z endstop? Or are you using a different solution
that wasn't listed? As one can work out what goes where Beth the several
pdf as to where they connect

**Reply by EKVoron24** [reply]
Hi,
Thanks for the reply.
I'm building a Voron 2.4 CoreXY printer and this has the x/y stops, hot end, hot end thermistor, hot end and part cooling fans as well as a couple of stepper motors on the moving gantry. I was looking to find where each of these connected to the Spider board as the wiring going through the BLTouch was a little different.
I have now worked out where each of the relevant connectors go and I'm now working through the initial testing.
Thanks again for your replies.
Regards
Eric

**Comment by cyberjak2k** [comment]
Glad to help!
